HHS/OPRE-OPS
# feat: ✨ adds AgreementBLIReviewTable

## What changed

- feat: ✨ adds AgreementBLIReviewTable
- feat: ✨ adds BLIReviewRow
- feat: ✨ handles state between action accordion and agreement budgetlines table

Does not include

- Table heading checkbox to select all
- e2e test updates

## Issue

#1558 

## How to test

1. Goto Agreements list
2. Choose Agreement 1 to review
3. see table row in disabled state
4. select DRAFT TO PLANNED
5. check table items
6. 💰profit
7. ♻️ Repeat for Agreement 9 but check for PLANNED TO EXECUTING
8. 💰💰profit

## Screenshots

<img width="1279" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/f19a07e9-ea0a-4c1c-9a23-732bf4a7473d">

## Definition of Done Checklist


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
